.github/workflows/pythonpackage.yml

```diff
@@ -59,8 +59,10 @@ jobs:
 
       - name: Install PyGeoprocessing
         run: |
-            conda install --file requirements.txt
-            conda install gdal==${{ matrix.gdal }} setuptools build
+            # Installing all of these in the same line will force conda to
+            # resolve all requirements at once, reducing the chance of DLL
+            # conflicts.
+            conda install --file requirements.txt gdal==${{ matrix.gdal }} setuptools build
```

Comment:
oh interesting! I'd be curious to read about how conda resolves dependencies when done all at once vs. sequentially. do you happen to have a reference for that?

Reply:
I don't have a documentation reference for it, but it does appear to be the case just from debugging this PR.  [Here's the latest build on `main`](https://pipelines.actions.githubusercontent.com/bGFNKdWfd0FvLn6TeVKOltVTPj0bb20eF1nGnpYGH1aTzpEgid/_apis/pipelines/1/runs/819/signedlogcontent/21?urlExpires=2022-03-24T23%3A21%3A07.3392228Z&urlSigningMethod=HMACV1&urlSignature=VLgds40U8QnPWloFlJDFKO%2B6wS%2FUKbkf9xcZsKGxX1Q%3D), where if you look at the logs you can see that the latest version of GDAL is being installed (3.4.2), but is then being downgraded to the build matrix-defined version of GDAL, 3.2.2.